ajax/question_condition.php

```diff
@@ -32,24 +32,16 @@
  */
 
 include ('../../../inc/includes.php');
-Session::checkRight("entity", UPDATE);
+Session::checkRight('entity', UPDATE);
 
 if (!isset($_REQUEST['plugin_formcreator_questions_id'])) {
    exit;
 }
-$questionId = (int) $_REQUEST['plugin_formcreator_questions_id'];
-if ($questionId == 0) {
-   $formId = (int) $_REQUEST['plugin_formcreator_forms_id'];
-} else {
-   $form = new PluginFormcreatorForm();
-   $form->getByQuestionId($questionId);
-   $formId = $form->getID();
-}
```

Comment:
$_REQUEST['plugin_formcreator_questions_id'] is always set when calling this entry point